20190411/MINUTES.md

```diff
@@ -0,0 +1,89 @@
+# Minutes
+
+Minutes from the meeting held on 2019/04/11.
+
+Participants:
+
+- ALBA: Zbigniew Reszela
+- DESY: Teresa Núñez
+- MAXIV: Antonio Milán, Juliano Murari, Mirjam Lindberg, Darren Spruce, Áureo Freitas
+- SOLARIS: Grzegorz Kowalski
+
+## Actions
+* define criteria for configuration tools comparison
+* use Slack a little and evaluate
+* create project for porting to Python 3
+
+## 1. Presentation: Actual state of DSConfig
+* `sardana-dsconfig` prepared specifically to handle Sardana
+* elements and their configuration stored in Excel file (not FODS)
+* separate sheets for different types of elements
+* dump live system configuration to json file
+* workflow: take existing config -> update from Excel file -> push to Tango
+* `json2tango` pushes json configuration to Tango DB
+* IDs for existing items are kept, new IDs are assigned by `sardana-dsconfig` script
+* separate tool for generating Excel files from json
+* Ansible module for populating Tango DB currently in development
```

Comment:
It is already developed and used for deploying Tango Device Servers, but not for Sardana ... yet.